# Results_scRNAseq_m1_internship

In this repository, I put all my results of my m1 internship on the re-evaluation of transcriptome data in single cells of early mouse embryo.

I use https://github.com/learogue/scRNAseq_project to construct a standard and test multiple parameters.

In `Results_standard`, there is plots on quality and filters of datasets. I can't put datas because of size. And there is also a jupyter notebook on the analysis of this standard with UMAP's, Volcano plot and differential expression of genes.

In `Results_EBs`, there is results of analysis with embryoid bodies. This embryoid bodies are different because of the inudction : pharmacologically or genetically.

In `Appendix_scripts`, there are others scripts that I made to create plots.

# References

References of the data that I used.

Deng, Q., Ramsköld, D., Reinius, B., and Sandberg, R. (2014). Single-Cell RNA-Seq Reveals Dynamic, Random Monoallelic Gene Expression in Mammalian Cells. Science 343, 193–196. https://doi.org/10.1126/science.1245316.

Chen, G., Schell, J.P., Benitez, J.A., Petropoulos, S., Yilmaz, M., Reinius, B., Alekseenko, Z., Shi, L., Hedlund, E., Lanner, F., et al. (2016). Single-cell analyses of X Chromosome inactivation dynamics and pluripotency during differentiation. Genome Res. 26, 1342–1354. https://doi.org/10.1101/gr.201954.115.

Mohammed, H., Hernando-Herraez, I., Savino, A., Scialdone, A., Macaulay, I., Mulas, C., Chandra, T., Voet, T., Dean, W., Nichols, J., et al. (2017). Single-Cell Landscape of Transcriptional Heterogeneity and Cell Fate Decisions during Mouse Early Gastrulation. Cell Reports 20, 1215–1228. https://doi.org/10.1016/j.celrep.2017.07.009.

Nowotschin, S., Setty, M., Kuo, Y.-Y., Liu, V., Garg, V., Sharma, R., Simon, C.S., Saiz, N., Gardner, R., Boutet, S.C., et al. (2019). The emergent landscape of the mouse gut endoderm at single-cell resolution. Nature 569, 361–367. https://doi.org/10.1038/s41586-019-1127-1.

Posfai, E., Schell, J.P., Janiszewski, A., Rovic, I., Murray, A., Bradshaw, B., Pardon, T., Bakkali, M.E., Talon, I., and Rossant, J. Defining totipotency using criteria of increasing stringency. https://doi.org/10.1101/2020.03.02.972893

Argelaguet, R., Clark, S.J., Mohammed, H., Stapel, L.C., Krueger, C., Kapourani, C.-A., Imaz-Rosshandler, I., Lohoff, T., Xiang, Y., Hanna, C.W., et al. (2019). Multi-omics profiling of mouse gastrulation at single-cell resolution. Nature 576, 487–491. https://doi.org/10.1038/s41586-019-1825-8.

Vrij, E.J., Scholte Op Reimer, Y.S., Fuentes, L.R., Guerreiro, I.M., Holzmann, V., Aldeguer, J.F., Sestini, G., Koo, B.-K., Kind, J., Van Blitterswijk, C.A., et al. (2022). A pendulum of induction between the epiblast and extra-embryonic endoderm supports post-implantation progression. Development 149, dev192310. https://doi.org/10.1242/dev.192310

Schumacher, S., Fernkorn, M., Marten, M., Chen, R., Kim, Y.S., Bedzhov, I., and Schröter, C. (2024). Tissue-intrinsic beta-catenin signals antagonize Nodal-driven anterior visceral endoderm differentiation. Nat Commun 15, 5055. https://doi.org/10.1038/s41467-024-49380-0.

